src/v1_2/mod.rs

```diff
@@ -195,9 +195,12 @@ pub struct Timings {
     pub dns: Option<f64>,
     #[serde(default = "default_fsize_maybe")]
     pub connect: Option<f64>,
-    pub send: f64,
-    pub wait: f64,
-    pub receive: f64,
+    #[serde(default = "default_fsize_maybe")]
+    pub send: Option<f64>,
+    #[serde(default = "default_fsize_maybe")]
+    pub wait: Option<f64>,
+    #[serde(default = "default_fsize_maybe")]
+    pub receive: Option<f64>,
```

Comment:
According to the HAR v1.2 spec, `send`, `wait` and `receive` are all required fields. They even explicitly write that they are **not** optional, see:

http://www.softwareishard.com/blog/har-12-spec/#timings

![image](https://github.com/user-attachments/assets/60fbbd13-6d98-49eb-bf59-3f0efc5ede04)